pkgs/applications/misc/textpieces/default.nix

```diff
@@ -0,0 +1,82 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python39
+, appstream-glib
+, desktop-file-utils
+, gtk3
+, gtksourceview5
+, gobject-introspection
+, meson
+, ninja
+, pkg-config
+, vala
+, wrapGAppsHook4
+, glib
+, gtk4
+, json-glib
+, libadwaita
+, libgee
+, wayland
+, nix-update-script
+}:
+let
+  pythonEnv = python39.withPackages (p: [ p.pyyaml ]);
```

Comment:
```suggestion
  pythonEnv = python3.withPackages (p: [ p.pyyaml ]);
```

Reply:
May I ask what your reasoning for changing python39 to python3 is?

Reply:
Right now python3 points to python39 and in the future if we change the default python version we do not want to cleanup explicit pints unless necessary.